### DISCUSSION TITLE
Admob sdk update

### DISCUSSION THREAD

**Discussion by merbin2012** [admob-plus/admob-plus#540] [Show and tell]
Currently, this plugin supports 20.6.0, but the higher version is Admob 21.3.0, so I just manually changed the value in the project.properties like below.
**com.google.android.gms:play-services-ads:20.6.0** to **com.google.android.gms:play-services-ads-lite:21.3.0** is this change is ok or not? After changing this value I am not faced any problems yet. But I want your clarification.

Thank you

**Comment by mujaffars** [comment]
@merbin2012 can you please tell how to do it from config.xml? Is it possible?

**Reply by merbin2012** [reply]
Platforms/Android/project.property file

**Comment by tomitank** [comment]
here is my clean only cordova approach, which is based on this repo:
https://github.com/tomitank/cordova-admob-tomitank

**Reply by merbin2012** [reply]
@tomitank what special feature is added than admob plus plugin?

**Reply by merbin2012** [reply]
@tomitank What special feature is added than admob plus? You just increase the sdk version, I'm sure we can't expect new features and future updates from you.  If it is no, add some extra functionality than this plugin like adaptive banner and add new methods other wise we won't use your new library. 

Thanks for your effort.

**Comment by tomitank** [comment]
@merbin2012 
Why i need to add new feature? For SDK update you need to change often the
code as well.
Ratson is not updated the code anymore. If you don't want to use my repo,
then pls don't use it :D
